applications/FemToDemApplication/custom_processes/generate_dem_process.cpp

```diff
@@ -168,4 +181,20 @@ int GenerateDemProcess::GetMaximumDEMId()
     return max_id;
 }
 
+/***********************************************************************************/
+/***********************************************************************************/
+
+int GenerateDemProcess::GetMaximumFEMId()
+{
+    int max_id = 0;
+    const auto it_FEM_node_begin = mrModelPart.NodesBegin();
+    // #pragma omp parallel for
```

Comment:
you can look in mortar_utilities for a OMP code to get the max and min

Reply:
I couldn't find the methods @loumalouomega  :S